<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wSuggestFiles</class>
 <widget class="QMainWindow" name="wSuggestFiles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">wSuggestFiles</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lblReport">
      <property name="text">
       <string>Selected link(s) for report:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pteReport">
      <property name="font">
       <font>
        <family>Consolas</family>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblFiles">
      <property name="text">
       <string>Please enter file URLs here:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pteFiles">
      <property name="font">
       <font>
        <family>Consolas</family>
       </font>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblNotes">
      <property name="text">
       <string>Optional: Notes</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pteNotes">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>18</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QCheckBox" name="cbxAcceptRules">
      <property name="text">
       <string>I have read and accept the rules.</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnRules">
        <property name="text">
         <string>Rules</string>
        </property>
        <property name="icon">
         <iconset resource="../../ressources.qrc">
          <normaloff>:/rec/data/icons/iconRule.svg</normaloff>:/rec/data/icons/iconRule.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnSend">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
        <property name="icon">
         <iconset resource="../../ressources.qrc">
          <normaloff>:/rec/data/icons/iconSend.svg</normaloff>:/rec/data/icons/iconSend.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>706</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
